src/main/java/org/example/TaskManager.java

```diff
@@ -0,0 +1,133 @@
+package org.example;
+import org.apache.commons.lang3.*;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.io.Writer;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Scanner;
+
+public class TaskManager {
+    static File file = new File("tasks.csv");
+    static String[][] tasks;
+    static final String[] OPTIONS = {"add", "remove", "list", "exit"};
+    public static void showMainMenu() {
+        System.out.println(ConsoleColors.BLUE + "\nPlease select an option: " + ConsoleColors.RESET);
+        for (String option : OPTIONS) {
+            System.out.println("-> " + option);
+        }
+        menuButtons();
+    }
+
+    public static void menuButtons() {
+        Scanner scanner = new Scanner(System.in);
+        String input = scanner.nextLine();
```

Comment:
CLEAN-CODE: promenne v metodach pokud to jde davat nemenne tedy final, vyhnes se omylu ze do promenne neco dvakrat prirazujes.
```suggestion
        final String input = scanner.nextLine();
```

Reply:
opraveno